shell/components/formatter/WorkloadDetailEndpoints.vue

```diff
@@ -27,17 +27,27 @@ export default {
 
         try {
           out = JSON.parse(this.value);
+
           out.forEach((endpoint) => {
             let protocol = 'http';
 
             if (endpoint.port === 443) {
               protocol = 'https';
             }
 
-            if (endpoint.addresses) {
+            const linkDefaultDisplay = endpoint.port ? `${ endpoint.port }${ endpoint.protocol }` : endpoint.protocol;
```

Comment:
Missing a slash between the port and protocol

Reply:
Thanks for catching that 🙏